### ISSUE TITLE
Fancy tabbar squashed when setting custom font

### ISSUE THREAD

**Issue by thomascft** [wez/wezterm#2399]
### What Operating System(s) are you seeing this problem on?

Linux Wayland

### Which Wayland compositor or X11 Window manager(s) are you using?

hyprland

### WezTerm version

20220807-113146-c2fee766

### Describe the bug

I'm getting squashed tabs when setting a custom font for my tabbar. I've tried `Victor Mono` and `Inter`. Below is a screenshot of the issue.
![08-13-22_00-17-01](https://user-images.githubusercontent.com/103956335/184471587-c9cde147-c6db-47af-b41a-2ee30c22ecff.png)

### To Reproduce

Set window_frame.font to value other than roboto

### Configuration

```
local wezterm = require("wezterm")

return {
        font = wezterm.font_with_fallback({{family="Victor Mono", weight="Medium"}, "FreeMono"}),
        font_size = 12,
        color_scheme = "tokyodark",
        window_background_opacity = 1,
        -- hide_tab_bar_if_only_one_tab = true,
        colors = {
                tab_bar = {
                        active_tab = {
                                bg_color = "#11121D",
                                fg_color = "#A0A8CD"
                        },
                        inactive_tab = {
                                bg_color = "#212234",
                                fg_color = "#A0A8CD"
                        },
                        inactive_tab_hover = {
                                bg_color = "#1A1B2A",
                                fg_color = "#A0A8CD"
                        },
                        new_tab = {
                                bg_color = "#212234",
                                fg_color = "#A0A8CD"
                        },
                        new_tab_hover = {
                                bg_color = "#11121D",
                                fg_color = "#A0A8CD"
                        },
                        inactive_tab_edge = "#212234"
                }
        },
        window_frame = {
                font = wezterm.font { family = 'Victor Mono', weight = 'Medium' },
                font_size = 12,
                active_titlebar_bg = "#212234",
                inactive_titlebar_bg = "#212234",
        },
}
```

**Comment by wez** [comment]
This should be fixed now in `main`.

It typically takes about an hour before fixes are available as nightly builds for all platforms.  Linux builds are the fastest to build and are often available within about 20 minutes.  Windows and macOS builds take a bit longer.

Please take a few moments to try out the fix and let me know how that works out. You can find the nightly downloads for your system in the [wezterm installation docs](https://wezfurlong.org/wezterm/installation.html).

If you prefer to use packages provided by your distribution or package manager of choice and don't want to replace that with a nightly download, keep in mind that you can download portable packages (eg: a `.dmg` file on macOS, a `.zip` file on Windows and an `.AppImage` file on Linux) that can be run without permanently installing or replacing an existing package, and can then simply be deleted once you no longer need them. 

If you are eager and can [build from source](https://wezfurlong.org/wezterm/install/source.html) then you may be able to try this out more quickly.

**Comment by thomascft** [comment]
Waiting on the actions workflow to complete and I'll give it a try!

**Comment by thomascft** [comment]
It's working great! Any Idea when this will be merged into stable?

**Comment by wez** [comment]
There's no stable branch; only `main` and releases cut from `main`. I'm considering making another release in the next couple of days.

**Comment by thomascft** [comment]
That answers my question. I was just wondering when it would be marked as a release.
